### PR TITLE
Updates to schemas and graphql-schemas

### PR DESCRIPTION
**Pull request by bkez322**
This is an attempt at getting the schemas defined in this repository to match those of a private "upstream" repository as much as possible, while still omitting the upstream's highly-implementation-specfic details (e.g. opinionated enum declarations, references to particular internal systems).

### CODE REVIEW

File: graphql-schemas/schema.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@
   - { name: kubeBinary, type: string, isRequired: true }
   - { name: pullRequestGateway, type: string }
   - { name: mergeRequestGateway, type: string }
-  - { name: saasDeployJobTemplate, type: string }
-  - { name: hashLength, type: int }
+  - { name: saasDeployJobTemplate, type: string, isRequired: true }
```
Comment (BumbleFeng) [inline graphql-schemas/schema.yml:12]: We could also loose these kinds of restrictions

You can simply remove all the changes in graphql-schemas/schema.yml that only add `isRequired: true` for the existing field.

File: schemas/app-sre/app-1.yml [hunk 2]
```diff
@@ -324,3 +329,9 @@ required:
 - labels
 - name
 - description
+- escalationPolicy
```
Comment (BumbleFeng) [inline schemas/app-sre/app-1.yml:332]: Same here.

Remove all the changes of adding required field for an existing schema file.

**Comment by bkez322** [comment]
I've finally come back to this, and I've updated schemas in a hopefully-agreeable manner. 

Prior to these changes, I could not run terraform integrations locally successfully. With these changes, I can run them successfully.

**Comment by bkez322** [comment]
I made the changes suggested by @BumbleFeng.

I tested that I was still able to successfully run local terraform integrations with these changes.

I'll go ahead and merge this.

**Pull request merged**